tests/foreman/longrun/test_oscap.py

```diff
@@ -386,20 +471,68 @@ def test_positive_oscap_run_via_ansible_bz_1814988(
 
     :BZ: 1814988
     """
+    contenthost = rex_contenthost
+    os_version = contenthost.os_version.major
+    distro = f'rhel{os_version}'
+    result = contenthost.register(
+        module_org,
+        None,
+        ak_name[distro],
+        target_sat,
+        ignore_subman_errors=True,
+        force=True,
+        insecure=True,
+    )
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
+    rhel_repo = rhel_repos[distro]
+    profile = profiles[distro]
+    if distro not in ('rhel7'):
+        contenthost.create_custom_repos(**rhel_repo)
+    else:
+        contenthost.create_custom_repos(**{distro: rhel_repo})
+
+    # Harden the client with DISA STIG security policy
+    contenthost.run('yum install -y scap-security-guide')
+    contenthost.run(
+        'oscap xccdf eval --remediate --profile xccdf_org.ssgproject.content_profile_stig '
+        '--fetch-remote-resources --results-arf results.xml '
+        f'/usr/share/xml/scap/ssg/content/ssg-{distro}-ds.xml',
+    )
+
+    # list existing content on Sat
+    existing_content_names = get_existing_content(target_sat)
+    # the content name depends on upload style (default vs file upload)
+    if (OSCAP_DEFAULT_CONTENT[f'{distro}_content'] not in existing_content_names) and (
+        f'{distro} content' not in existing_content_names
+    ):
+        upload_scap_content_for_host(target_sat, contenthost, module_org)
+
+    # re-list after content upload
+    existing_content_names = get_existing_content(target_sat)
+    # find out content name for client version
+    assert (OSCAP_DEFAULT_CONTENT[f'{distro}_content'] in existing_content_names) or (
+        f'{distro} content' in existing_content_names
+    ), f'scap content not found for {distro}, existing content: {existing_content_names}'
+    content = (
+        OSCAP_DEFAULT_CONTENT[f'{distro}_content']
+        if (OSCAP_DEFAULT_CONTENT[f'{distro}_content'] in existing_content_names)
```

Comment:
Given the repetition of `OSCAP_DEFAULT_CONTENT[f'{distro}_content']` and how long the lines are I'd recommend assigning that result to a variable.